library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js

```diff
@@ -224,6 +261,28 @@
         });
     });
 
+    portal.hideShowAlerts = function () {
+        $.ajax({
+            url: '/direct/portal/hideShowBullhornAlert.html',
+            dataType: 'html',
+            cache: false,
+            success: function (data) {
+                alertHidden = data === 'true';
+                toggleHideShowText();
+                updateCounts();
+            }
+        });
+    };
+
+    var toggleHideShowText = function () {
+        var hideShowLink = $('#Mrphs-hideshow-alerts');
+        if (alertHidden) {
```

Comment:
`alertHidden` is undefined in this method. Did you mean to pass it into the method on line 271 above?